src/components/common/AppHeader/AppHeaderSearchInput/AppHeaderSearchInput.jsx

```diff
@@ -1,53 +1,54 @@
 import React, { useState } from 'react';
 import PropsTypes from 'prop-types';
 import { connect } from 'react-redux';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
 
 import Input from '../../Typography/Input';
 import MemberModule from '../../../../services/member';
 import Icon from '../../Typography/Icon';
-import { faSearch } from '@fortawesome/free-solid-svg-icons';
 
 const { getSearchText } = MemberModule.selectors;
-const {setSearchText} = MemberModule.actions;
+const { setSearchText: setSearchTextToStore } = MemberModule.actions;
 
 /**
  * This component is search field for the member list.
- * @param props
- * @returns {*}
+ * @param {Object} props
+ * @return {*}
  * @constructor
  */
 const AppHeaderSearchInput = (props) => {
   const [searchText, setSearchText] = useState(props.searchText);
 
   const onChangeInput = (event) => {
-    const {target: {value}} = event;
+    const { target: { value } } = event;
 
     // This check is for when user search a string and click on search.
     // Filtered data is displayed. But when user clear search box he
     // does not have to click search again so we set the empty search
     // to load the default list.
     if (value.length === 0 && (value !== props.searchText)) {
```

Comment:
Can be simplified as:
```suggestion
    if (!value.length && (value !== props.searchText)) {
```

Reply:
Done.